CMakeLists.txt

```diff
@@ -77,6 +77,7 @@ set(LIBRARY_SOURCES
     Src/PBREffect.cpp
     Src/PBREffectFactory.cpp
     Src/pch.h
+    Src/pch.cpp
```

Comment:
I had trouble getting a clean build to work with PCH because it has to build pch.cpp first... How do you get it to do that?

Reply:
I just wiped my `build` sub-directory and rebuilt and VS builds `pch.cpp` first for me (I am using VS2019 16.4.2).  I assumed it just knows to do that in order to make pre-compiled headers work.  I am using CMake 3.16.1, which is not the one bundled with VS.